package.json

```diff
@@ -70,6 +92,23 @@
   ],
   "license": "MIT",
   "main": "bin/semantic-release.js",
+  "nyc": {
+    "include": [
+      "src/**/*.js"
+    ],
+    "reporter": [
+      "json",
+      "text",
+      "html"
+    ],
+    "all": true
+  },
+  "prettier": {
+    "printWidth": 120,
+    "singleQuote": true,
+    "bracketSpacing": false,
+    "trailingComma": "es5"
+  },
```

Comment:
IMO it's better to put prettier config directly inside eslint config. we're using `eslint-plugin-prettier`, right?

Reply:
No because prettier config is read by editor tools and by the `prettier` command itself.
Actually it work the other way around, it's `eslint-plugin-prettier` that read the prettier config. See prettier/eslint-plugin-prettier#55

Reply:
> Actually it work the other way around, it's eslint-plugin-prettier that read the prettier config. See prettier/eslint-plugin-prettier#55

yeah, it's added in v2.3.0. but it still can read the config directly if we put the config in `rules` field

Reply:
Prettier config in `prettier` property, Eslint config in `eslintConfig` property seems more clear to me. Not sure what would be the point of moving it around.
But thanks for the feedback :)